behaviors/forms.py

```diff
@@ -32,7 +32,7 @@ def __init__(self, request=None, *args, **kwargs):
     def save(self, commit=True):
         obj = super(EditoredModelForm, self).save(commit=False)
 
-        if self.request is not None and self.request.user.is_authenticated():
+        if self.request is not None and self.request.user.is_authenticated:
```

Comment:
this now won't work for django 1.8 which is still supported until April, we need to add a `compat.py` module for this, ill add one in for ya